datalab/settings.py

```diff
@@ -137,6 +137,8 @@ def get_list_from_env(variable, default=None):
 # Datalab Archive
 ARCHIVE_API = os.getenv('ARCHIVE_API', 'https://archive-api.lco.global')
 ARCHIVE_API_TOKEN = os.getenv('ARCHIVE_API_TOKEN')
+if not ARCHIVE_API_TOKEN:
+    print("WARNING: ARCHIVE_API_TOKEN is missing from the environment.")
```

Comment:
Until we change these print statements to logging, this is fine. We probably also want to to `exit()` if there is no ARCHIVE_API_TOKEN found.